client/src/pages/ErrorPage/ErrorPage.module.scss

```diff
@@ -0,0 +1,44 @@
+.container {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    & > img {
+        width: 50vw;
+    }
+    & > h2 {
+        font-size: 25px;
+        line-height: 30px;
+        margin-bottom: 30px;
```

Comment:
no need to change the typography styles of our h2 elements,  since we have default styles in our theme for h2 and h3. just use typography variant = 'h2' or 'h3' and let the default styles take place in order to achieve a uniform design across our app.